.circleci/config.yml

```
@@ -502,7 +502,7 @@ jobs:
   e2e_tests_adapters:
     <<: *e2e-executor
     docker:
-      - image: cypress/browsers:node-18.16.1-chrome-114.0.5735.133-1-ff-114.0.2-edge-114.0.1823.51-1
+      - image: cypress/browsers:node-18.17.19-chrome-114.0.5735.133-1-ff-114.0.2-edge-114.0.1823.51-1
```

```suggestion
      - image: cypress/browsers:node-18.20.3-chrome-125.0.6422.141-1-ff-126.0.1-edge-125.0.2535.85-1
```

Per https://hub.docker.com/r/cypress/browsers/tags?name=node-18

---

thanks